lib/pacemaker/pcmk_sched_allocate.c

```diff
@@ -788,22 +807,8 @@ pcmk__schedule_actions(pe_working_set_t *data_set, xmlNode *xml_input)
 
     crm_trace("Create transition graph");
     pcmk__create_graph(data_set);
-
-    crm_trace("=#=#=#=#= Summary =#=#=#=#=");
-    crm_trace("\t========= Set %d (Un-runnable) =========", -1);
```

Comment:
There can't possibly be a user out there that's grepping for these strings in their log file, can there?

Reply:
Log messages of any kind don't have any sort of guarantees (we change them pretty frequently), but trace logs in particular are purely for development